website/app/styles/pages/home.scss

```diff
@@ -1,36 +1,173 @@
-  // TODO check with heather what she wants to do with the vertical padding (responsive)
-  padding: 16px var(--doc-page-stage-gutter-small);
+  // visual top padding is controlled by the "hero" element, so we set 0 here
+  padding: 0 var(--doc-page-stage-gutter-small) 64px;
+  color: var(--doc-color-white);
 
-  @include breakpoint.medium () {
-    padding: 32px var(--doc-page-stage-gutter-medium);
+  @include breakpoint.medium() {
+    padding: 0 var(--doc-page-stage-gutter-medium) 64px;
   }
 
-  @include breakpoint.large () {
-    padding: 64px var(--doc-page-stage-gutter-medium);
+  @include breakpoint.large() {
+    // 1216px by design
+    max-width: calc(1216px + 2 * var(--doc-page-stage-gutter-large));
+    padding: 0 var(--doc-page-stage-gutter-large) 128px;
   }
 }
 
+
+// hero
+
+.doc-page-home__hero {
+  @include breakpoint.small() {
+    --padding-top: 260px; // 200 of decoration on top + 60px of negative space
+    --padding-bottom: 100px;
+    --title-font-size: 2.625rem; // 42px
+    --tagline-font-size: 1rem; // 16px
+  }
+
+  @include breakpoint.medium() {
+    --padding-top: 120px; // decoration is below the content
+    --padding-bottom: 100px;
+    --title-font-size: 3.75rem; // 60px
+    --tagline-font-size: 1.25rem; // 20px
+  }
+
+  @include breakpoint.large() {
+    --padding-top: 300px; // decoration is below the content
+    --padding-bottom: 180px;
+    --title-font-size: 4.375rem; // 70px
+    --tagline-font-size: 1.25rem; // 24px
+  }
+
+  display: flex;
+  flex-direction: column;
+  padding: var(--padding-top) 0 var(--padding-bottom);
+
+  h1 {
+    @include doc-font-family("gilmer");
+    margin: 0;
+    color: var(--doc-color-white);
+    font-weight: 500;
+    font-size: var(--title-font-size);
+    line-height: 1;
+  }
+
+  p {
+    @include doc-font-family("gilmer");
+    max-width: 350px;
+    margin: 2em 0;
+    color: var(--doc-color-gray-500);
+    font-size: var(--tagline-font-size);
+    line-height: 1.55;
+
+    @include breakpoint.medium-and-above() {
+      max-width: 600px;
+    }
+  }
+}
+
+.doc-page-home__hero-links {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 10px 40px;
+}
+
+// cards
+
 .doc-page-home__cards {
-  margin-top: -132px;
-  padding: 32px;
-  background-color: #fff;
-  border-radius: 8px;
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 16px;
+  margin: 0;
+  padding: 0;
+
+  @include breakpoint.medium() {
+    grid-template-columns: repeat(2, 1fr);
+    gap: 24px;
+  }
+
+  @include breakpoint.large() {
+    grid-template-columns: repeat(4, 1fr);
+    gap: 32px;
+  }
+}
+
+.doc-page-home__card {
+  position: relative; // needed so the link will fill the whole card
+  display: flex;
+  flex-direction: column;
+  padding: 18px;
+  list-style: none;
+  border: 1px solid var(--doc-color-gray-200);
+  border-radius: 3px;
+
+  @include breakpoint.medium() {
+    min-height: 200px;
+    padding: 24px;
+  }
+
+  @include breakpoint.large() {
+    min-height: 280px;
```

Comment:
Is the `min-height` here to account for different lengths of content within the cards (with placeholder content for now)? When there isn't a lot of content there's extra white space at the bottom.

Reply:
+1. These shouldn't have as much extra white space on the bottom as they do now. The bottom padding should be 32px.

I'm not as familiar with grid's capabilities yet...is it possible to have the description container be the section that gets taller so that the CTA links can all be aligned (like in this example)?
<img width="630" alt="Screen Shot 2022-12-14 at 5 58 38 PM" src="https://user-images.githubusercontent.com/8553306/207754620-389febcb-5a56-475f-a2ae-7213315e98e0.png">

Reply:
fixed in https://github.com/hashicorp/design-system/pull/807/commits/b3b254f69abd6eb56076236bcc35f1acdb0f75e3 (I was looking at the initial designs, not at the ones in the handover)